newspack-newsletters.php

```diff
@@ -55,3 +55,5 @@
 require_once NEWSPACK_NEWSLETTERS_PLUGIN_FILE . '/includes/class-newspack-newsletters-quick-edit.php';
 require_once NEWSPACK_NEWSLETTERS_PLUGIN_FILE . '/includes/class-newspack-newsletters-embed.php';
 require_once NEWSPACK_NEWSLETTERS_PLUGIN_FILE . '/includes/class-newspack-newsletters.php';
+
+\Newspack\Newsletters\Subscription_Lists::init();
```

Comment:
Until we fully change the strategy on how things are required and initialized, can we keep the `init()` inside the class file as we do for all other classes in the plugin?

Reply:
Any particular reason for that?

I think we should start moving forward doing this the "right" way... meaning relying on the autoloader (so the classes are onlyu loaded if needed) and initializing things from outside the file. But a huge refactor might be traumatic, so we can so it step by step...